prov/efa/src/rdm/rxr_env.c

```diff
@@ -99,6 +102,24 @@ void rxr_env_define_use_device_rdma()
 		}
 	}
 
+	return use_device_rdma;
+}
+
+
+/** @brief: Define FI_EFA_USE_DEVICE_RDMA as a configuration parameter
```

Comment:
nit: Inconsistent `@brief` indentation

Reply:
Did you want the @brief to be on the next line?

Reply:
Let's make it consistent with the rest of the code.